Frends.SFTP.WriteFile/Frends.SFTP.WriteFile/Frends.SFTP.WriteFile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.ComponentModel;
+using Renci.SshNet;
```

Comment:
Add `using Frends.SFTP.WriteFile.Definitions;` after changing namespace in the parameter files.

Reply:
Fixed